datafusion/physical-plan/src/insert.rs

```diff
@@ -74,10 +74,14 @@ pub trait DataSink: DisplayAs + Debug + Send + Sync {
     ) -> Result<u64>;
 }
 
+#[allow(dead_code)]
+#[deprecated(since = "38.0.0", note = "Use [`DataSinkExec`] instead")]
+type FileSinkExec = DataSinkExec;
```

Comment:
I think this type alias needs to be marked public to enable downstream code referencing the old name to continue to work. That should also eliminate the dead code warning I believe.

Reply:
Ah yeah, will fix

Reply:
Fixed